### PR TITLE
Add upcoming semester dates to calendar

### PR DESCRIPTION
**Pull request by Robert27**
Please double check the dates before approving

https://www.thi.de/fileadmin/daten/studienangelegenheiten/Allgemeines/Semestertermine_Studium_WS2425_und_SS25_01.pdf

### CODE REVIEW

**Review by McBaumwolle — APPROVED** [review]

File: rogue-thi-app/data/calendar.json [hunk 1]
```diff
@@ -10,219 +10,307 @@
...
   {
     "name": {
-      "de": "Vorlesungsfrei",
-      "en": "No lectures"
+      "de": "Prüfungsanmeldung",
+      "en": "Registration for exams"
+    },
+    "begin": "2024-04-23",
+    "end": "2024-05-02"
+  },
```
Comment (BuildmodeOne) [inline rogue-thi-app/data/calendar.json:33]: These are all in the past.

File: rogue-thi-app/data/calendar.json [hunk 2]
```diff
@@ -10,219 +10,307 @@
...
+    "end": "2024-05-21"
+  },
+  {
+    "name": {
+      "de": "Vorlesungsfrei (Fronleichnam)",
+      "en": "No lectures (Corpus Christi)"
     },
-    "begin": "2023-10-03"
+    "begin": "2024-05-30"
+  },
```
Comment (BuildmodeOne) [inline rogue-thi-app/data/calendar.json:64]: Also in the past.

**Review by BuildmodeOne — CHANGES_REQUESTED** [review]
The confirmation of studies are still missing

File: rogue-thi-app/data/calendar.json [hunk 3]
```diff
@@ -1,74 +1,11 @@
 [
-  {
```
Comment (BuildmodeOne) [inline rogue-thi-app/data/calendar.json:2]: We should leave that here

**Review by BuildmodeOne — CHANGES_REQUESTED** [review]

File: rogue-thi-app/data/calendar.json (hunk 3 above)
Comment (Robert27) [reply rogue-thi-app/data/calendar.json:2]: indeed

**Review by BuildmodeOne — APPROVED** [review]
lgtm

**Pull request merged**